src/openforms/formio/formatters/tests/test_default_formatters.py

```diff
@@ -0,0 +1,51 @@
+import json
+import os
+
+from django.test import TestCase
+
+from ..service import format_value
+
+FILES_DIR = os.path.join(
+    os.path.dirname(os.path.abspath(__file__)),
+    "files",
+)
+
+
+def load_json(filename: str):
+    with open(os.path.join(FILES_DIR, filename), "r") as infile:
+        return json.load(infile)
+
+
+class DefaultFormatterTestCase(TestCase):
+    def test_formatters(self):
+        all_components = load_json("all_components.json")["components"]
+        data = load_json("all_components_data.json")
+        expected = {
```

Comment:
Since we do not have formatters for `map` and `signature` yet, should I leave them out for now?

Reply:
yeah you can leave them out for now